reference/api/documents.mdx

```diff
@@ -180,6 +275,10 @@ If you send an already existing document (same [document id](/learn/core_concept
 
 To completely overwrite a document, check out the [add or replace documents route](/reference/api/documents#add-or-replace-documents).
 
+<Capsule intent="note">
+Meilisearch orders documents in the order they were inserted into the database.
```

Comment:
Perhaps add a short sentence describing why this is important?

Reply:
@irevoire 
Do you have anything on this?

Reply:
Huum, no, it's not important, I just noticed a sentence was wrong in the spec and fixed it, but actually, it doesn't really have anything to do with the initial PR :sweat: 
I don't think we should document it

Reply:
I'll remove it then. Thanks!